### PR TITLE
GitHub Actions part 1: Add GHA script by converting Travis CI script into it

### PR DESCRIPTION
**Pull request by xor-freenet**
Please merge with `--ff-only` into `next`.

There are **none** of my other open PRs included in this branch, they are
on a different branch.  
Hence I would kindly ask you to merge this one first so there is a working CI
system at Freenet's repo more soon.

---

**NOTICE**: For security, before merging this please go to the `Settings` tabs
of Freenet's and your own WebOfTrust repository and set:
`Actions / Workflow permissions: Read repository contents permission.`
Then click `Save` below that.
In other words: It should not have write-access.
Please confirm by commenting that you did remove the write access.

### CODE REVIEW

**Comment by ArneBab** [comment]
Thank you for the warning about setting read-only. I set that now.
